docs/get_started.rst

```diff
@@ -1,17 +1,388 @@
+
+    $ cci service connect github
+
+Verify the GitHub service is connected by running ``cci service list``:
+
+.. image:: images/service-list.png
+
+Once you've configured the ``github`` service it will be available to **all** CumulusCI projects.
+
+Services are stored in the global CumulusCI keychain by default.
+
+
+
+Work on an Existing CumulusCI Project
 -------------------------------------
 
-Start a new CumulusCI project
------------------------------
+Follow this section if there's an existing CumulusCI project on GitHub that you'd like to work on.
+
+.. note:: CumulusCI does not support projects stored on other Git hosts such as BitBucket or GitLab at this time.
+
+There are three preliminary steps for working on an existing CumulusCI project. You need to:
+
+* `Install CumulusCI`_
+* `Install Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+* `Clone the Project's GitHub Repository <https://docs.github.com/en/free-pro-team@latest/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories>`_
+
+You can change directories into the project's root directory and begin executing ``cci`` commands. For example, ``cci project info`` displays information about the project:
+
+.. code-block:: console
+
+    $ cd cumulusci-test
+
+    $ cci project info
+    name: CumulusCI Test
+    package:
+        name: CumulusCI Test
+        name_managed: None
+        namespace: ccitest
+        install_class: None
+        uninstall_class: None
+        api_version: 33.0
+    git:
+        default_branch: main
+        prefix_feature: feature/
+        prefix_beta: beta/
+        prefix_release: release/
+        release_notes:
+            parsers:
+                1:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Critical Changes
+                2:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Changes
+                3:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubIssuesParser
+                    title: Issues Closed
+                4:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: New Metadata
+                5:
+                    class_path: cumulusci.tasks.release_notes.parser.GithubLinesParser
+                    title: Deleted Metadata
+        repo_url: https://github.com/SFDO-Tooling/CumulusCI-Test
+    test:
+        name_match: %_TEST%
+
+
+
+Starting a New CumulusCI Project
+--------------------------------
+
+There are three preliminary steps for starting a brand new CumulusCI project. You need to:
+
+#. `Install CumulusCI`_
+#. `Install Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
+#. `Install the Salesforce CLI <https://developer.salesforce.com/tools/sfdxcli>`_
```

Comment:
Similar comment as above in Work On an Existing CumulusCI Project. It also looks like this one is an ordered list and the one in Existing is unordered. They should be consistent, unless order doesn't matter in Existing and it does matter here?

Reply:
Order doesn't matter here.
```suggestion
There are three preliminary steps for starting a brand new CumulusCI project. You need to:

* `Install CumulusCI`_
* `Install Git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`_
* `Install the Salesforce CLI <https://developer.salesforce.com/tools/sfdxcli>`_
```